content/docs/iac/concepts/glossary.md

```diff
@@ -10,6 +10,7 @@ menu:
 aliases:
 - /docs/reference/glossary/
 - /docs/concepts/glossary/
+- /learn/glossary/
```

Comment:
This is actually incorrect -- there are two glossaries (unfortunate as that is) -- one in the docs, and one in tutorials. This line actually belongs in `content/tutorials/glossary`. I've done that in https://github.com/pulumi/docs/pull/12826/commits/f43d573ea6547dd04290bedd6ba5a3d9cc34f42c.

```suggestion
```